css/index.css

```diff
@@ -1,3 +1,281 @@
+  letter-spacing: 1px;
+  margin-bottom: 15px;
+}
+
+/* Your code starts here! */
+
+.body-header {
+  display: flex;
+  flex-direction: column;
+  margin: 0px 50px;
+}
+
+
+
+.nav-logo-container, .header-section {
+  display: flex;
+  flex-direction: row;
+}
+
+.nav-logo-container {
+  justify-content: space-between;
+}
+
+
+@media (max-width: 800px) {
+    .nav-logo-container {
+        flex-direction: column;
+        align-items: center;
+        margin-right: 30px;
+       height: 119px;
+    }
+}
+
+@media (max-width: 500px) {
+    .nav-logo-container {
+        flex-direction: column;
+        align-items: center;
+        margin-right: 30px;
+       height: 100px;
+    }
+}
+
+.header-section {
+  justify-content: space-around;
+  margin: 60px 0px;
+  padding-bottom: 20px;
+}
+
+@media (max-width: 800px) {
+    .header-section {
+       padding-bottom: 0px;
+    
+      }
+  }
+
+  @media (max-width: 500px) {
+    .header-section {
+       margin-top: 350px;
+    
+      }
+  }
+
+#header-nav {
+  display: inline-flex;
+  width: 75%;
+  flex-direction: row;
+  margin: 40px 0px;
+  justify-content: space-between;
+}
+ @media (max-width: 800px){
+     #header-nav{
+order: 1;
+width: 100%;
+justify-content: space-between;
+
+     }
+ }
+
+ @media (max-width: 500px) {
+    #header-nav {
+        flex-direction: column;
+        align-items: center;
+
+    }
+}
+
+#header-nav a {
+	text-decoration: none;
+  color: lightgray;
+}
+
+
+
+@media (max-width: 800px) {
+    #header-nav a {
+         width: 50px;
+      }
+  }
+
+  
+@media (max-width: 500px) {
+    #header-nav a {
+        padding-bottom: 20px;
+        margin-bottom: 40px;
+      }
+  }
+
+.logo-container {
+  display: inline-flex;
+  width: 24%;
+  flex-direction: row;
+  justify-content: flex-end;
+  margin: 30px 0px 30px 50px;
+}
+
+
+.h1-button-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+h1 {
+  font-size: 80px;
+  text-align: center;
+}
+
+.header-button {
+  border: 1px solid black;
+  font-size: 20px;
+  padding: 5px 40px;
+}
+
+.snippet-image-container {
+  display: flex;
+  flex-direction: row;
+}
+
+@media (max-width: 800px) {
+    .snippet-image-container{
+        display: none;
+    } 
+}
+
+#main {
+  display: flex;
+  flex-direction: column;
+  border-top: 1px solid black;
+  margin: 0px 50px 50px;
+  padding-top: 50px;
+}
+
+@media (max-width: 800px) {
+   #main {
+       margin: 0px 25px 25px;
+       padding-top: 40px;
+     
+      }
+  }
+
+#footer {
+  border-top: 1px solid black;
+  display: flex;
+  flex-direction: column;
+  margin: 0px 50px 50px;
+}
+
+.main-container-top, .main-container-middle, .main-container-bottom {
+  display: inline-flex;
+  flex-direction: row;
+}
+
+.features-text, .about-text {
+  width: 90%;
+}
+
+.main-container-middle {
+  justify-content: center;
+  margin: 50px 0px;
+}
+
+
+
+.section-features {
+  width: 50%;
+  padding: 0px 10px;
+}
+
+.section-about {
```

Comment:
For all these sections I would change to `width: 100%` in the 500px media query to make stack better in the mobile view